lib/elixir/lib/float.ex

```diff
@@ -149,6 +149,11 @@ defmodule Float do
   @spec floor(float, precision_range) :: float
   def floor(number, precision \\ 0)
 
+  def floor(number, 0) when is_float(number) do
+    int = :erlang.floor(number)
```

Comment:
This seems like a perfect use of pipes, is there any reason not to use them here?

```elixir
number
|> :erlang.floor()
|> :erlang.float()
```